http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/GceRuntimeMonitor.scala

```diff
@@ -55,52 +63,90 @@ class GceRuntimeMonitor[F[_]: Parallel](
       startMonitoring <- nowInstant
       monitorContext = MonitorContext(startMonitoring, runtimeAndRuntimeConfig.runtime.id, traceId, action)
       _ <- Timer[F].sleep(config.initialDelay)
-      poll = googleComputeService
-        .pollOperation(googleProject, operation, config.pollingInterval, config.pollCheckMaxAttempts)
-        .compile
-        .lastOrError
-      op <- poll
+      haltWhenTrue = (Stream
+        .eval(clusterQuery.getClusterStatus(runtimeAndRuntimeConfig.runtime.id).transaction.map { newStatus =>
+          if (action == RuntimeStatus.Deleting) // Deleting is not interruptible
+            false
+          else {
+            newStatus != Some(action) && newStatus == Some(
+              RuntimeStatus.Deleting
+            ) // Interrupt Stopping if new Deleting request comes in
+          }
+        }) ++ Stream.sleep_(5 seconds)).repeat
```

Comment:
this should probably be in a config

Reply:
there are way too many intervals like this...I think I added these in configs in early PRs..but I'm leaning toward having them in code now